pinc/wordcheck_engine.inc

```diff
@@ -343,7 +343,7 @@ function get_bad_words_via_external_checker($input_words_w_freq, $languages)
                     "list",
                     "--prefix=" . escapeshellarg($aspell_prefix),
                     "-d " . escapeshellarg($dict_file),
-                    "--encoding " . escapeshellarg($charset),
+                    "--encoding=UTF-8",
```

Comment:
`"--encoding=UTF-8"` looks correct. Was the old version with `"--encoding UTF-8"` wrong, but still working?

Reply:
Both work, aspell accepts args with or without `=`s: `--param value` or `--param=value`